### ISSUE TITLE
EVA evidence to link to MEF2A in coronary heart disease and two children terms

### ISSUE THREAD

**Issue by deniseOme** [opentargets/platform#841]
PMID: 15496429 and PMID: 15958500 describe rs121918531 and rs121918529 as evidence for an association between MEF2A (ENSG00000068305) and myocardial infarction (EFO_0000612) and coronary artery disease (EFO_0000378).

These are the issues:

1) we don't link MEF2A with CAD based on genetic associations (only via text mining evidence):
https://www.targetvalidation.org/evidence/ENSG00000068305/EFO_0000378. The SNPs should have been used as genetic evidence for this association as well.

2) the association between MEF2A and CHD is indirect with genetic evidence from EVA propagated from CAD to CHD (child-parent). On the UI, we should not have CHD in the "Diseases" column in the genetic associations table; we should list MI and/or CAD instead.

![Screen Shot 2020-02-13 at 10 33 37](https://user-images.githubusercontent.com/6472381/74430325-914a3380-4e54-11ea-8d61-457ea12d7379.png)

This gives the wrong impression to users that CHD is associated with MEF2A directly, where it should be an indirect association.

**Comment by AsierGonzalez** [comment]
This issue seems to be caused by wrong mapping of ClinVar conditions to EFO terms. Both variants are annotated with the `coronary artery disease/myocardial infarction` condition in ClinVar (see [rs121918531](https://www.ncbi.nlm.nih.gov/clinvar/RCV000009507/) and [rs121918529](https://www.ncbi.nlm.nih.gov/clinvar/RCV000009505/)) but then are mapped to `coronary heart disease` (http://www.ebi.ac.uk/efo/EFO_0001645) and `myocardial infarction` (http://www.ebi.ac.uk/efo/EFO_0000612") in our evidence:

```
{
  "id": "http://www.ebi.ac.uk/efo/EFO_0001645",
  "name": "coronary heart disease",
  "source_name": "coronary artery disease/myocardial infarction"
}
{
  "id": "http://www.ebi.ac.uk/efo/EFO_0000612",
  "name": "myocardial infarction",
  "source_name": "coronary artery disease/myocardial infarction"
}

```
I will ask EVA to look into it.

**Comment by AsierGonzalez** [comment]
This has been fixed for 20.04. EVA now have a [file to track problematic mappings](https://docs.google.com/spreadsheets/d/1m4ld3y3Pfust5JSOJOX9ZmImRCKRGi-fGYj_dExoGj8/edit#gid=0) that they correct manually prior to submission.
